sql/sql_servers.cc

```diff
@@ -112,6 +112,54 @@ static void init_servers_cache_psi_keys(void)
 }
 #endif /* HAVE_PSI_INTERFACE */
 
+
+bool trt_table_type_is_correct(TABLE *t)
```

Comment:
Don't see the reason why there is second function dedicated just for type checking.